benchmarks/calc_signal_benchmark.py

```diff
@@ -42,9 +42,9 @@
 # Defining defaults
 ###################
 
-_LRES = [-1,-3,0]
-_LLOS = [1,3,0]
-_LT = [1,3,0]
+_LRES = [-3,-3,0]
+_LLOS = [5,5,0]
+_LT = [3,3,0]
```

Comment:
this is probably shouldn't be part of the PR. We can leave it if you want @Didou09, but for cleaningness maybe you want to remove it ?

Reply:
Hum, not sure, it's good to have easy access to typical becnhmark input

Reply:
ok